dev/preview/install-k3s-kubeconfig.sh

```diff
@@ -24,17 +28,33 @@ do
     esac
 done
 
-if [[ "${BRANCH}" == "" ]]; then
-    VM_NAME="$(preview-name-from-branch)"
-else
-    VM_NAME="$(preview-name-from-branch "$BRANCH")"
+if [ -z "${VM_NAME:-}" ]; then
+  if [[ "${BRANCH}" == "" ]]; then
+      VM_NAME="$(preview-name-from-branch)"
+  else
+      VM_NAME="$(preview-name-from-branch "$BRANCH")"
+  fi
 fi
 
-echo "Installing context from VM: $VM_NAME"
-
 K3S_CONTEXT="${VM_NAME}"
 K3S_ENDPOINT="${VM_NAME}.kube.gitpod-dev.com"
 
+# If we can access the k3s cluster, there's nothing to do
+if kubectl --context="${K3S_CONTEXT}" auth can-i get secrets 2>&1 1>/dev/null ;then
+  exit 0
```

Comment:
Shouldn't this be exit 1 then? If we invoke this script and we can't access the cluster I would expect that to be an error ☺️

Reply:
We only want to handle the case here if we have access to exit early. Otherwise if we don't have access or if it's an error continue with the execution and try to get the credentials.

Reply:
Ah, right, that makes sense ✅